### ISSUE TITLE
Evaluate Cross-Encoder

### ISSUE THREAD

**Issue by r1ckC139** [UKPLab/sentence-transformers#1488]
Hi there,
I have some questions about evaluate Cross-Encoder:
1. When call model.fit() the evaluator parameter must be a class of SentenceEvaluator (sentence_transformers.evaluation.BinaryClassificationEvaluator for example). Besides, i can see another evaluation is sentence_transformers.cross_encoder.evaluation.CEBinaryAccuracyEvaluator. Why don't use this evaluator in model.fit() instead SentenceEvaluator, i think SentenceEvaluator just for Bi-Encoder.
2. Is there any way to print out loss and accuracy every logging steps during training cross-encoder ?
